resources/views/index.blade.php

```diff
@@ -1,23 +1,24 @@
 @extends('layouts.app')
 
+@section('main-class', '-archive')
+
 @section('content')
+  @parent
   @include('partials.page-header')
 
-  @if (! have_posts())
-    <x-alert type="warning">
-      {!! __('Sorry, no results were found.', 'sage') !!}
-    </x-alert>
+  @if (have_posts())
+    <div class="listing">
+      @while(have_posts()) @php(the_post())
+        <x-post-tile class="listing__tile" :post-id="get_the_ID()"/>
+      @endwhile
+    </div>
 
-    {!! get_search_form(false) !!}
+    <div class="pagination">
+      {!! get_the_posts_navigation() !!}
+    </div>
+  @else
+    <div class="error-message">
+      {{ __('Sorry, no results were found.', 'wps') }}
```

Comment:
@coditive-tomek-g @coditive-przemek-h  I assume that we will change `wps` to `firestarter`?

Reply:
@coditive-pawel-m - I think we should use `sage` as a text-domain, @coditive-przemek-h do you agree? If yes I'll change it.